docs/Authenticating-Users-via-SAML.md

```diff
@@ -172,6 +172,14 @@ If you need a very different parsing of the SAML tokens than what is done at `or
     saml.logout.url=<idp specific logout URL, e.g. https://idp.logoutpage.com >
     
 
+Some IDPs (e.g. Azure Active Directory) cache user data for more than 2 hours causing cbioportal to complain that the authentication statement is too old to be used. You can fix this problem by setting `forceAuthN` to true. Below is an examle how you can do this with the properties. You can choose any binding type you like. `bindings:HTTP-Redirect` is given just as an example.
```

Comment:
MInor typo s/examle/example/.

Also, should we prefix the comment with a warning icon like the previous custom scenarios?

![image](https://user-images.githubusercontent.com/366003/176468459-0008ccf0-5800-4b53-9e6d-e9849e9be405.png)

Reply:
Thanks @n1zea144, I've added the warning sign as you suggested.

Reply:
I still see a spelling error above (examle instead of example)

Reply:
Ah, sorry. I haven't seen your comment on that. I've force pushed the change.